rust-crates/symblib/src/gosym/mod.rs

```diff
@@ -540,3 +540,55 @@ fn range_rel2abs(base: VirtAddr, rng: Range<raw::TextStartOffset>) -> Range<Virt
         end: base.wrapping_add(rng.end.0),
     }
 }
+
+impl symbconv::PointResolver for GoRuntimeInfo<'_> {
+    fn symbols_for_pc(&self, pc: VirtAddr) -> symbconv::Result<Vec<symbconv::ResolvedSymbol>> {
```

Comment:
This currently doesn't support resolving inline functions. That's OK for an initial PR IMO, `symbconv` currently doesn't support it either, but we should add an explaining comment at least:

```
// NOTE: this is currently doesn't support inline functions
```

On the positive side, it's a lot easier to implement the resolving logic for point lookups than for transformation into `symbfile` range format, if we want to do that in the future.

Reply:
added the note with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/380/commits/ee8050ce8546e6e87f78c59f3d668e6a90dbd7bf